docs/understanding-airbyte/airbyte-protocol.md

```diff
@@ -460,7 +458,6 @@ This is the new pattern for referring to a stream. As structs are updated, they
 ```yaml
   StreamDescriptor:
     type: object
-    additionalProperties: true
```

Comment:
For this `airbyte-protocol` doc, since these are describing actual models from https://github.com/airbytehq/airbyte/blob/master/airbyte-protocol/protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml, they should probably actually reflect the content that's in the real models